Document login multi factor authentication (MFA) (#152)
N8-CIR-Bede/documentation: usage/index.rst

batch scheduler, and so demanding applications should be submitted to it
(see "Running Jobs").

.. _usage-mfa:

Multi Factor Authentication
---------------------------

.. note::

   We are gradually rolling out MFA, so this may not yet apply to your account.

When connecting to the login nodes, the system has to validate your
identity. In order to help keep accounts secure, Bede employs a

> technique called Multi Factor Authentication, where something you

Reviewer: ```suggestion
technique called Multi Factor Authentication (MFA), where something you
```

know (your password), is combined with something you have (an app on
a mobile device, such as your phone).

When you first login to Bede, or after a password reset, you will be prompted
to:

1. Change your password
2. Import an MFA token into an app on your mobile device

Instead of `Password`, to login after this you will be prompted for:

> - `First Factor` - this is your password

Reviewer: In RST single backticks are "interpreted text", which is generally combined with an extension to have concrete meaning. 

I'm assuming you meant to mark it as inline ala markdown? in which case it's double backticks. Alternatively `*` or `**` for emphasized (italics) and strongly emphasized (bold) text. 

```suggestion
- ``First Factor`` - this is your password
```

> - `Second Factor` - this is a one-time password code provided by the token in your app

Reviewer: ```suggestion
- ``Second Factor`` - this is a one-time password code provided by the token in your app
```

Mobile apps that can be used to import an MFA token include Microsoft Authenticator,

> Google Authenticator, and others. When the token is generated on Bede, it will

Reviewer: It might be worth adding the required protocol for "others" to make it more clear. I assume it's TOTP, but potentially HOTP based on Google Auth support.

```suggestion
Google Authenticator, and other TOTP authentication clients. When the token is generated on Bede, it will
```

If it is TOTP the relevenat RFC could be mentioned too, but TOTP will be enough info (if this is needed at all).

> display a QR code, which can then be imported into the app (e.g. select `+` to add

Reviewer: ```suggestion
display a QR code, which can then be imported into the app (e.g. select ``+`` to add
```

> an account and then `Scan a QR code` or `Other account` to scan the QR code).

Reviewer: ```suggestion
an account and then ``Scan a QR code`` or ``Other account`` to scan the QR code).
```

If you are unable to use an MFA app, for example if you do not have a mobile
device, please contact `support <../faq/#faq-helpsupport>`__ for options.

> 

Author: ```suggestion

If you have lost your password or MFA token, please use EPCC's SAFE system to request a password reset for your Bede login account, which we normally aim to process within a working day.

```

Regarding adding something on what to do if an MFA token is lost, it was implicit that a password reset would do the job, but would be better made explicit.
